src/quo2/components/tags/status_tags.cljs

```diff
@@ -50,42 +50,48 @@
                     :style {:padding-left 5
                             :color text-color}} label]]])))
 
-(defn positive [size theme label]
-  [base-tag {:size size
+(defn- positive
+  [size theme label]
+  [base-tag {:size             size
              :background-color colors/success-50-opa-10
-             :icon :verified
-             :border-color colors/success-50-opa-20
-             :text-color (if (= theme :light) colors/success-50
-                             colors/success-60)
-             :label (or label (i18n/label :positive))}])
+             :icon             :verified
+             :border-color     colors/success-50-opa-20
+             :label            (or label (i18n/label :positive))
+             :text-color       (if (= theme :light) colors/success-50
+                                   colors/success-60)}])
 
```

Comment:
hi @ilmotta, why these changes needed?

Reply:
@Parveshdhull, what happened here is that I had improved the status tag component to accept a label, but someone also fixed it and merged before I did. So this morning I fixed the conflicts, but these changes remained after my rebase (map keys were sorted). They're harmless though